DTAConfig/OptionPanels/DisplayOptionsPanel.cs

```diff
@@ -812,7 +812,7 @@ public override bool Save()
             IniSettings.Translation.Value = (string)ddTranslation.SelectedItem.Tag;
 
             // copy translation files to the game directory
-            foreach (TranslationGameFile tgf in ClientConfiguration.Instance.TranslationGameFiles)
+            ClientConfiguration.Instance.TranslationGameFiles.AsParallel().ForAll(tgf => 
```

Comment:
Will this wait for all the tasks to complete?

Reply:
Yes, as previous code.